docs/docs/adrs/adr-020-cutomizable_slashing_and_jailing.md

```
@@ -0,0 +1,185 @@
+For example, if we take `$600` as the cost of running a validator node, a budget of `$3000` will be sufficient to cover the cost of four validators running such a consumer chain and have `$150` profit per validator as incentive.
+In practice, this can be implemented via the per-consumer-chain commission rate that allows validators to have different commission rates on different consumer chains. 
+
+### Implementation
+
+The implementation of this feature involves the following steps:
+
+* Add the `InfractionParameters` to `MsgCreateConsumer`.
+* On slashing events (for either downtime or double signing infractions), use the corresponding `slash_fraction` set by the consumer chain.
+* On jailing events (for either downtime or double signing infractions), use the corresponding `jail_duration` set by the consumer chain.
+* Cryptographic equivocation evidence received for PoR chains results in the misbehaving validators only being tombstoned and not slashed.
+* (Optional) Add the `InfractionParameters` to `MsgUpdateConsumer`, i.e., allow consumer chains to update the slashing and jailing parameters, but the changes will come into effect after a period equal to the staking module's unbonding period elapses to allow for validators to opt out.   
```

Markdownlint

Expected: 0 or 2; Actual: 3
Trailing spaces

(MD009, no-trailing-spaces)